main.tex

```diff
@@ -1,23 +1,23 @@
-\documentclass[titlepage]{book}
-\usepackage[utf8]{inputenc}
-
+\documentclass[letterpaper,american,final]{memoir}
```

Comment:
I'm not entirely sold on using memoir

Reply:
this class is a flexible version of book, you choose